src/templates/index.html

```diff
@@ -26,13 +26,14 @@ <h3 class="greeting-content">Quote of the day</h3>
 <script>
     function heartClicked(button) {
         var icon = button.querySelector('i');
+        
+        // Toggle the heart icon appearance
         icon.classList.toggle('far');
         icon.classList.toggle('fas');
-
-        // If the quote is hearted
+        
+        // Only send a request to the dedicated heart endpoint when it's a like action
```

Comment:
NIT: it's the way it should be. A little bit odd to stress it :)

 // ~Only s~Send a request to the ~dedicated~ heart endpoint when it's a like action